python/pythonlab/neighborhood/neighborhood/support/position.py

```diff
@@ -0,0 +1,4 @@
+class Position:
+    def __init__(self, x: int, y: int):
+        self.x = x
+        self.y = y
```

Comment:
it looks like in Java Lab we also [track direction](https://github.com/code-dot-org/javabuilder/blob/main/org-code-javabuilder/validation/src/main/java/org/code/validation/Position.java#L4) as part of position, do we not need that or is that going to be in a follow-up?

Reply:
Ah yes! I meant to include this in my PR description!
I initially included direction. However, I was puzzled why the case `DIRECTION` was not handled `PainterTracker.java`'s `trackEvent`.

I saw that position object's direction is not ever referenced. The [direction from `detail`](https://github.com/code-dot-org/javabuilder/blob/0ad7f3de06201e90685359ba45ffb0c537d2f1f9/org-code-javabuilder/validation/src/main/java/org/code/validation/support/PainterTracker.java#L40) is the one to determine the position's x and y updated values. 


So I plan to add `direction` as a `PainterTracker` instance variable. Hope this makes sense!

Reply:
The public-facing [PainterLog](https://github.com/code-dot-org/javabuilder/blob/0ad7f3de06201e90685359ba45ffb0c537d2f1f9/org-code-javabuilder/validation/src/main/java/org/code/validation/PainterLog.java#L79-L85) class does return `position`, which includes direction. I wonder if it is needed in validation?

Reply:
We could add `get_direction` to the `PainterLog` class. But definitely defer to curriculum what they would prefer.

Reply:
Updated so that direction is part of `Position` so that we can keep the API as similar as possible. Thanks!